lightly/api/api_workflow_datasets.py

```diff
@@ -1,7 +1,13 @@
+from os import access
 from typing import List
 
 from lightly.openapi_generated.swagger_client.models.create_entity_response import CreateEntityResponse
 from lightly.openapi_generated.swagger_client.models.dataset_create_request import DatasetCreateRequest
+
+from lightly.openapi_generated.swagger_client.models.shared_access_config_create_request import SharedAccessConfigCreateRequest
+from lightly.openapi_generated.swagger_client.models.shared_access_config_data import SharedAccessConfigData
+from lightly.openapi_generated.swagger_client.models.shared_access_type import SharedAccessType
+
```

Comment:
Why are these additional imports necessary?

Reply:
I assume that they are a leftover from moving the code to a new file.

Reply:
Oh, my bad. Good catch!